internal/api/grpc_service.go

```diff
@@ -268,6 +279,14 @@ func (s *GrpcService) GetDeviceMakeByName(ctx context.Context, in *p_grpc.GetDev
 		result.TokenId = deviceMake.TokenID.Uint64()
 	}
 
+	result.ExternalIdsTyped = []*p_grpc.ExternalID{}
+	for _, ei := range deviceMake.ExternalIdsTyped {
+		result.ExternalIdsTyped = append(result.ExternalIdsTyped, &p_grpc.ExternalID{
+			Vendor: ei.Vendor,
+			Id:     ei.ID,
+		})
+	}
+
```

Comment:
it looks like you're doing this very similar operation in 3 in this same file and 1 place in common/utils.go, consider refactoring